packages/office-ui-fabric-react/src/components/List/examples/List.Grid.Example.tsx

```diff
@@ -75,65 +66,49 @@ const classNames: IListGridExampleClassObject = mergeStyleSets({
   },
 });
 
-const ROWS_PER_PAGE = 3;
-const MAX_ROW_HEIGHT = 250;
-
-export class ListGridExample extends React.Component<IListGridExampleProps> {
-  private _columnCount: number;
-  private _columnWidth: number;
-  private _rowHeight: number;
-  private _items: IExampleItem[];
-
-  constructor(props: IListGridExampleProps) {
-    super(props);
-    this._items = props.items || createListItems(5000);
-  }
+const onRenderCell = (item: any, index: number | undefined): JSX.Element => (
```

Comment:
Same with other `item: any` in list examples
```suggestion
const onRenderCell = (item: IExampleItem, index: number | undefined): JSX.Element => (
```